apps/HowtoAddNewApp.md

```diff
@@ -0,0 +1,185 @@
+# How to add new application
+
+Five files, **source file**, [**Kconfig**](#kconfig), [**Kconfig_ENTRY**](#kconfig_entry), [**Make.defs**](#makedefs) and [**Makefile**](#makefile) are mandatory to add a new application.
```

Comment:
I think it should be rephrased like this .
The following files are required to integrate a new application with the TizenRT build system :-
* Kconfig
* Kconfig_ENTRY
* Make.defs
* Makefile